custom/icds_reports/const.py

```diff
@@ -18,6 +18,13 @@
                'kafka-ucr-main']
 
 
+class NavigationSections:
```

Comment:
Why is this defined on the server side when it's only used in js?

Reply:
I also wondered the best way to do this... What's a good way to have a shared, JS-only, complex data structure that is dependent on django variables / template tags? Happy to switch to something else if there's a better option. I debated between this one and using an `{% include %}` tag with a hand-coded javascript dict but neither one seemed particularly great.

Reply:
This is the right thing to do if there's a django dependency, I just don't see what that dependency is. Right now it looks like translations and `static`, neither of which is a blocker for defining this in js.

Reply:
Ah, I think I mistakenly assumed there wasn't an easy way to do `static` in JS. If you have an example handy I think it should be swappable to do that (I'll probably put in a follow up PR once I have a chance to confirm that it won't ever need anything else).

Reply:
For `static` I think it's fine to hard code the real path, like `static/icds_reports/mobile/images/gray-info.png`. We do this in styles, like https://github.com/dimagi/commcare-hq/pull/26003/files#diff-b4971aa19aea8783d68c88ba1bfa8348R199

Reply:
interesting. I think that would break / be painful if we switched to a CDN or used something like ManifestStorage down the line. but I guess if there's a precedent then we're not worried about that?

Reply:
(in my head the use of a relative path in that example made it less bad... though i think it would still break on Manifest... and i had made a mental note to revisit it at a later point...)

Reply:
the fact that I also had to mock this in a test is another argument for moving to JS, so will likely do that in a followup PR soonish

Reply:
I do think hard-coding paths feels a little funny, but yeah, there's precedent, so if we switch to a system where that doesn't work, we'll need to come up with an alternative.

ManifestStaticFilesStorage is clever enough to find and replace references to the path names that it updates, which somewhat ironically is one of the reasons it's non-trivial for us to switch over to it (a number of our bower dependencies have references to non-existent files, which Manifest can't handle, generally tests with fake image paths in CSS). My gut is we're more likely to switch to a front-end asset manager like webpack, but I don't know if/how hard-coded paths will be an issue there.